apidoc/Titanium/UI/WebView.yml

```diff
@@ -170,6 +170,22 @@ methods:
     since: {android: "8.1.0", iphone: "2.0.0", ipad: "2.0.0"}
     platforms: [android, iphone, ipad]
 
+  - name: loadFile
```

Comment:
Is this for loading html files? Or for other resources?

Reply:
Both.

Reply:
All html files should be handled via "url" which is already pretty smart handling different types of urls. In additions, assets should be loaded through the bridge between the WKWebView and the web-view handler.